src/components/simple-components/TextComponents.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled, { css } from 'styled-components';
+
+import { COLORS } from '../../constants';
+
+// This styling isn't final; just for testing and to demonstrate how we can use styled-components.
+const TextCss = css`
+    color: ${COLORS.BLUE_DARK};
+`;
```

Comment:
This `TextCss` bit is an example of a way to reuse CSS across components. I'm just using color as an example, but we would likely put things like fonts here.

Chances are we will actually have multiple reusable CSS blocks, like `HeaderCss` that will be applied to all headers.